packages/@aws-cdk/aws-ecs/lib/container-definition.ts

```diff
@@ -589,6 +602,56 @@ function renderUlimit(ulimit: Ulimit): CfnTaskDefinition.UlimitProperty {
     hardLimit: ulimit.hardLimit,
   };
 }
+/**
+ * The details of a dependency on another container in the task definition.
+ *
+ * @see https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_ContainerDependency.html
+ */
+export interface ContainerDependency {
+  /**
+   * The container to depend on.
+   */
+  readonly container: ContainerDefinition,
+  /**
+   * The dependency condition of the container. Valid values are ContainerDependencyCondition.START, ContainerDependencyCondition.COMPLETE,
```

Comment:
What is the dependency condition? Can you copy & paste from AWS docs?

Reply:
This is a copy & paste from the docs :) Added some more of my own context.